flink-python/tox.ini

```diff
@@ -28,7 +28,7 @@ whitelist_externals=
     /bin/bash
 deps =
     pytest
-    grpcio>=1.3.5,<=1.14.2
+    grpcio>=1.3.5,<=1.26.0
```

Comment:
1.3.5 -> 1.22.0?

Reply:
Good catch! I checked that the minimum version could be 1.17.0 and have updated the PR.